hypothesis-python/src/hypothesis/internal/conjecture/choice.py

```diff
@@ -234,9 +298,10 @@ def choice_to_index(choice, kwargs):
         raise NotImplementedError
 
 
-def choice_from_index(index, ir_type, kwargs):
+def choice_from_index(index: int, ir_type: str, kwargs: ChoiceKwargsT) -> ChoiceT:
```

Comment:
`ir_type: Literal["integer", "boolean", "bytes", "string", "float"]`?  Might even make sense to have a `ChoiceTName: "TypeAlias" = ...` somewhere.

Reply:
we already have `IRTypeName`, I just forgot about it! Have renamed + moved that to choice.py